packages/manager/modules/vrack/src/dashboard/vrack.controller.js

```diff
@@ -1395,6 +1435,16 @@ export default class VrackMoveDialogCtrl {
     return formattedService;
   }
 
+  static getOvhCloudConnectServerNiceName(service) {
+    const formattedService = {
+      ...service,
+      id: service.uuid,
+      niceName: service.uuid,
```

Comment:
```suggestion
      displayName: service.uuid,
```


I'll let you judge the relevance of the suggestion ;)

Reply:
I am not sure either because I need to check this with the PO. I will come back.